requirements.txt

```diff
@@ -1,4 +1,5 @@
-pbr>=0.10
+importlib
```

Comment:
The default version of python on freebsd is 2.7.

I think we do not need this.